build/ci.bazelrc

```diff
@@ -20,8 +20,13 @@ build:ci --color=yes
 build:ci --terminal_columns=100
 
 build:ci --config=v8-codegen-opt
-build:ci --remote_download_minimal --disk_cache=~/bazel-disk-cache
 test:ci --test_output=errors
+build:ci --disk_cache=~/bazel-disk-cache
+
+# normal CI build doesn't need outputs downloaded
+build:ci --remote_download_outputs=minimal
```

Comment:
I think it's safer to not set any default setting (so that we implicitly default to toplevel as of Bazel 7) and only opt in to minimal for the test job. I'm not sure if the npm publish/npm-types CI jobs rely on using toplevel